subsys/bluetooth/host/att.c

```diff
@@ -943,7 +920,7 @@ static void att_req_send_process(struct bt_att *att)
 			continue;
 		}
 
-		if (bt_att_chan_req_send(chan, req) >= 0) {
+		if (!bt_att_chan_req_send(chan, req)) {
```

Comment:
The API doesn't return a boolean, and using the negation operator makes it look like you're looking for an error when it's in fact indicative of success. I'd therefore recommend `== 0` for clarity.